Frey/data/migrations/20231118184700_add_platforms_tables.php

```diff
@@ -0,0 +1,39 @@
+<?php
+// @codingStandardsIgnoreFile
+use Phinx\Migration\AbstractMigration;
+
+require_once __DIR__ . '/../../src/helpers/BootstrapAccess.php';
+
+class AddPlatformsTables extends AbstractMigration
+{
+    public function up()
+    {
+        $this->_genTenhouAccounts();
+        $this->_genMajsoulAccounts();
+    }
+
+    protected function _genTenhouAccounts()
+    {
+        $table = $this->table('tenhou_platform_accounts');
+        $table
+            ->addColumn('tenhou_id', 'string', ['limit' => 255, 'null' => false])
+            ->addColumn('person_id', 'integer')
+            ->addIndex('tenhou_id', ['name' => 'person_tenhou_platform_idx'])
+            ->addForeignKey('person_id', 'person')
+            ->save();
+    }
+
+    protected function _genMajsoulAccounts()
+    {
+        $table = $this->table('majsoul_platform_accounts', ['id' => false, 'primary_key' => ['account_id', 'nickname']]);
+        $table
+            ->addColumn('nickname', 'string', ['limit' => 255, 'null' => false])
+            ->addColumn('person_id', 'integer')
+            ->addColumn('friend_id', 'integer', ['null' => false])
+            ->addColumn('account_id', 'integer', ['null' => false])
```

Comment:
tbh, I'd just add these several fields to `person` table so it would be simpler to manage things. The table is not big (~thousands of rows on main instance), so it should be safe.

Reply:
useless tenhou_platform_accounts removed, so table majsoul_platform_accounts still here because in person this  columns become nullable and impossible guarantee uniqueness (need multiple keys [null, null] => no way to create unique constraint)